components/FilterControls.tsx

```diff
@@ -169,6 +169,26 @@ export function FilterControls({
           Resolved
         </button>
 
+        <button
+          onClick={() =>
+            setExtraFilters({
+              ...extraFilters,
+              readyToResolve: false,
+              resolvingSoon: false,
+              unresolved: !extraFilters.unresolved,
+            })
+          }
+          className={clsx(
+            "btn hidden @xs:block",
```

Comment:
On mobile this makes it too wide for the screen:

<img width="457" alt="image" src="https://github.com/user-attachments/assets/f6c88bae-9174-42cf-80ae-728ec2301c57">

We should make this hidden `@sm:block` or maybe `@md:block` or lg/xl, not sure (you'll need to manually test to see what fits nicely). Or we could just have it appear in the extra buttons shown when the Filter button is selected.

Reply:
Looks like it has to be xl

Reply:
...maybe I spoke to soon, going to try a few more things

Reply:
Sounds good!